vagrant/src/main/vagrant/files/install_brooklyn.sh

```diff
@@ -48,45 +48,53 @@ fi
 if [ ! "${INSTALL_FROM_LOCAL_DIST}" == "true" ]; then
   if [ ! -z "${BROOKLYN_VERSION##*-SNAPSHOT}" ] ; then
     # url for official release versions
-    BROOKLYN_URL="https://www.apache.org/dyn/closer.lua?action=download&filename=brooklyn/apache-brooklyn-${BROOKLYN_VERSION}/apache-brooklyn-${BROOKLYN_VERSION}-bin.tar.gz"
-    BROOKLYN_DIR="apache-brooklyn-${BROOKLYN_VERSION}-bin"
+    BROOKLYN_URL="https://www.apache.org/dyn/closer.lua?action=download&filename=brooklyn/apache-brooklyn-${BROOKLYN_VERSION}/apache-brooklyn-${BROOKLYN_VERSION}.noarch.rpm"
   else
     # url for community-managed snapshots
-    BROOKLYN_URL="https://repository.apache.org/service/local/artifact/maven/redirect?r=snapshots&g=org.apache.brooklyn&a=brooklyn-dist&v=${BROOKLYN_VERSION}&c=dist&e=tar.gz"
-    BROOKLYN_DIR="brooklyn-dist-${BROOKLYN_VERSION}"
+    BROOKLYN_URL="https://repository.apache.org/service/local/artifact/maven/redirect?r=snapshots&g=org.apache.brooklyn&a=rpm-packaging&v=${BROOKLYN_VERSION}&c=noarch&e=rpm"
```

Comment:
`rpm-packaging` seems to be available for [releases](https://repository.apache.org/content/groups/public/org/apache/brooklyn/rpm-packaging/) but not for [snapshots](https://repository.apache.org/content/groups/snapshots/org/apache/brooklyn/rpm-packaging/).

Reply:
Good catch! That's odd, the docs says the reverse. I think @richardcloudsoft is looking at building RPM and DEB for snapshot versions